BoostTestAdapter/Utility/CommandLine.cs

```diff
@@ -64,14 +77,20 @@ public static IEnumerable<string> SplitCommandLine(string commandLine)
 
             return commandLine.Split(c =>
             {
-                if (c == '\\' && !isEscaping) { isEscaping = true; return false; }
+                if ((c == '\\') && (!isEscaping))
```

Comment:
[style, minor] I find the added parentheses here superfluous, even more so on the second operand.